replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+
+```
+
+> compared to the grocery store.
+
+```{r echo = FALSE}
+clean_1b %>%
+  filter(split == 1, resort == 0) %>%
+  summarise(mean.beer = mean(beer), CI.low = mean(beer) -  (1.96 * Sem(beer)), CI.high = mean(beer) + (1.96 * Sem(beer))) %>%
+  round(digits = 2)
+#1.96 * standard error gives 95% CI in one direction
+#this calculates high-income WTP beer price at grocer - summarise to display in print-out
+
+
+clean_1b_high_grocer <- clean_1b %>%
+  filter(split == 1, resort == 0) %>%
+  mutate(mean.beer = mean(beer), CI = (1.96 * Sem(beer))) %>%
+  distinct(mean.beer, CI)
+#generates mean and CI for high income participants in the grocer scenario - to use in figure 1b
+```
+
+> But lower income participants' WTP did not differ significantly between beer from the resort, 
+
+```{r echo = FALSE}
+clean_1b %>%
+  filter(split == 0, resort == 1) %>%
+  summarise(mean.beer = mean(beer), CI.low = mean(beer) -  (1.96 * Sem(beer)), CI.high = mean(beer) + (1.96 * Sem(beer))) %>%
+  round(digits = 2)
+#1.96 * standard error gives 95% CI in one direction
+#this calculates low income WTP beer price at resort
+
+clean_1b_low_resort <- clean_1b %>%
+  filter(split == 0, resort == 1) %>%
+  mutate(mean.beer = mean(beer), CI = (1.96 * Sem(beer))) %>%
+  distinct(mean.beer, CI)
+#generates mean and CI for low income participants in the resort scenario - to use in figure 1b
+```
+
+> and beer from the store.
+
+```{r echo = FALSE}
+clean_1b %>%
+  filter(split == 0, resort == 0) %>%
+  summarise(mean.beer = mean(beer), CI.low = mean(beer) -  (1.96 * Sem(beer)), CI.high = mean(beer) + (1.96 * Sem(beer))) %>%
+  round(digits = 2)
+#1.96 * standard error gives 95% CI in one direction
+#this calculates low income WTP beer price at grocery store
+
+clean_1b_low_grocer <- clean_1b %>%
+  filter(split == 0, resort == 0) %>%
+  mutate(mean.beer = mean(beer), CI = (1.96 * Sem(beer))) %>%
+  distinct(mean.beer, CI)
+#generates mean and CI for low income participants in the grocer scenario - to use in figure 1b
+```
+
+
+> The interaction between income and context was significant F(1,145) = 11.18, p < 0.01, eta squared partial = 0.07.
+
+```{r echo = FALSE}
+
+summary(aov_1b <- aov(beer ~ resort * split, data = clean_1b))
+
+temp_eta <- etaSquared(aov(beer ~ resort * split, data = clean_1b)) %>%
+  .[3,2] %>%
+  round(digit = 2)
+print(c("partial eta squared:", temp_eta), quote = FALSE)
+
+```
+
+> The interaction was also significant when income was treated as a continuous variable beta = 2.27, t(145) = 2.17, p < 0.05.
+
+```{r echo = FALSE}
+lm(scale(beer) ~ scale(resort) + scale(logses) + scale(resort * logses), data = clean_1b) %>%
+  tidy() %>%
+  filter(term == "scale(resort * logses)") %>%
+  mutate(beta = round(estimate, digit = 2),
+         p = round(p.value, digit = 4))  %>%  
+        select(term, beta, p)
+
+```
+
+
+```{r echo = FALSE}
+title = factor(c("Lower Income", "Higher Income"), levels = c("Lower Income", "Higher Income"))
```

Comment:
I'm confused as to why are you using " = " here instead of "<-" which you also do above